checks/devnet/devnet-construction.json

```diff
@@ -18,9 +18,9 @@
         "prefunded_accounts": [
             {
                 "account_identifier": {
-                    "address": "erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx"
+                    "address": "erd1ldjsdetjvegjdnda0qw2h62kq6rpvrklkc5pw9zxm0nwulfhtyqqtyc4vq"
                 },
-                "privkey": "b8ca6f8203fb4b545a8e83c5384da033c415db155b53fb5b8eba7ff5a039d639",
+                "privkey": "3e4e89e501eb542c12403fb15c52479e8721f2f4dedc3b3ef0f3b47b37de006c",
```

Comment:
private keys in config? Do we really need this? Why?

Reply:
We cannot use Rosetta's [`generate_key()`](https://www.rosetta-api.org/docs/rosetta_dsl_file.html#example-dsl-file) to generate keys in a given shard (whether actual shard or projected shard), therefore we have provide it both in `*.ros` and `*.json` config files. There are more like placeholders (even though I've prefunded these placeholder addresses, actually, on devnet) - these placeholders are generally replaced when testing.